metadata-ingestion/src/datahub/ingestion/source/neo4j/neo4j_source.py

```diff
@@ -113,78 +94,64 @@ def create(cls, config_dict: Dict, ctx: PipelineContext) -> "Neo4jSource":
         config = Neo4jConfig.parse_obj(config_dict)
         return cls(config, ctx)
 
-    def get_field_type(self, attribute_type: Union[type, str]) -> SchemaFieldDataType:
-        type_class: type = _type_mapping.get(attribute_type, NullTypeClass)
-        return SchemaFieldDataType(type=type_class())
-
-    def get_schema_field_class(
-        self, col_name: str, col_type: str, **kwargs: Any
-    ) -> SchemaFieldClass:
-        if kwargs["obj_type"] == self.NODE and col_type == self.RELATIONSHIP:
-            col_type = self.NODE
-        else:
-            col_type = col_type
-        return SchemaFieldClass(
-            fieldPath=col_name,
-            type=self.get_field_type(col_type),
-            nativeDataType=col_type,
-            description=col_type.upper()
-            if col_type in (self.NODE, self.RELATIONSHIP)
-            else col_type,
-            lastModified=AuditStampClass(
-                time=round(time.time() * 1000), actor="urn:li:corpuser:ingestion"
-            ),
+    def get_field_type_string(self, attribute_type: Union[type, str]) -> str:
+        """Convert Neo4j attribute type to DataHub string representation."""
+        type_key = (
+            attribute_type if isinstance(attribute_type, str) else str(attribute_type)
+        )
+        return _STRING_TYPE_MAPPING.get(type_key, "string")
```

Comment:
Can we have it as a const or define a list of DataHub-supported types at a shared place?